hedera-node/hedera-mono-service/src/main/resources/evm-addresses-blocklist.txt

```diff
@@ -0,0 +1,299 @@
+7f109a9e3b0d8ecfba9cc23a3614433ce0fa7ddcc80f2a8f10b222179a5a80d6
```

Comment:
Where do all these addresses come from?  Is there a separate document/spreadsheet/script? (If someone wants to know why an address is blocked is there a way to answer that?) If there is a separate document how is it kept in sync with this (and vice versa)?  Either way, would it be a good idea to have a 2nd column here of some kind of description of the addresses meaning?

Reply:
Yes, there is a separate doc in Notion with the initial list of accounts, but once this PR is merged, I think the file introduced here should be the source of truth and any other doc should be based on its contents.

I agree that another column could be useful and currently in the Notion doc we have separate sections for the different entries (e.g. Hedera Local Node addresses, Hardhat addresses and Test accounts used by hardhat), so I can add those probably?

Also after I add them do you think I should include those in the memo of the created accounts instead of having the same generic memo for all accounts?

Reply:
Added an additional `memo` column to the file.
@Nana-EC do you think using the account memo for this purpose is appropriate?

Reply:
Memo in file is fine.
Should make it optional i.e. doesn't fail if empty